packages/itwinui-react/src/core/Tile/TileNew.tsx

```diff
@@ -0,0 +1,177 @@
+  // SvgCheckmark,
+  // LinkAction,
+  PolymorphicComponentProps,
+  // useSafeContext,
+  // supportsHas,
+  PolymorphicForwardRefComponent,
+} from '../utils';
+import '@itwin/itwinui-css/css/tile.css';
+// import { DropdownMenu } from '../DropdownMenu';
+// import { IconButton } from '../Buttons';
+// import { ProgressRadial } from '../ProgressIndicators';
+// import Tile from '.';
+
+//<Tile>
+//  <Tile.Thumbnail>
+//  <Tile.Name>
+//  <Tile.Content>  //has meta data
+//  <Tile.Buttons>
+//</Tile>
+
+// ----------------------------------------------------------------------------
+// Main Tile component
+
+const TileComponent = React.forwardRef((props, ref) => {
+  const { as: Element = 'div', className, status, ...rest } = props;
+  return (
+    <Element
+      className={cx(
+        'iui-tile',
+        {
+          //   'iui-folder': variant === 'folder',
+          //   'iui-new': isNew,
+          //   'iui-selected': isSelected,
+          //   'iui-actionable': isActionable,
+          [`iui-${status}`]: !!status,
+          //   'iui-loading': isLoading,
+        },
+        className,
+      )}
+      ref={ref}
+      {...rest}
+    />
+  );
+}) as PolymorphicForwardRefComponent<'div', TileOwnProps>;
+
+type TileOwnProps = {
+  /**
+   * Status of the tile.
+   */
+  status?: 'positive' | 'warning' | 'negative';
+};
+
+// ----------------------------------------------------------------------------
+// Tile.Thumbnail component
+
+const TileThumbnail = React.forwardRef((props, ref) => {
+  const {
+    as: Element = 'div',
+    className,
+    thumbnail,
+    leftIcon,
+    rightIcon,
+    badge,
+    ...rest
+  } = props;
+  return (
+    <Element
+      className={cx(
+        'iui-tile-thumbnail',
+        {
+          // 'iui-tile-thumbnail-picture': typeof picture === 'string',
+          // 'iui-tile-thumbnail-type-indicator': leftIcon,
+          //   'iui-selected': isSelected,
+          //   'iui-actionable': isActionable,
+          // [`iui-${status}`]: !!status,
+          //   'iui-loading': isLoading,
+        },
+        className,
+      )}
+      style={
+        {
+          // backgroundImage: `url(${picture})`,
+          // typeof picture === 'string' ? `url(${picture})` : undefined,
+        }
+      }
+      ref={ref}
+      {...rest}
+    >
+      {thumbnail && typeof thumbnail === 'string' ? (
+        <div
+          className='iui-tile-thumbnail-picture'
+          style={{ backgroundImage: `url(${thumbnail})` }}
+        />
+      ) : thumbnail && (thumbnail as JSX.Element).type === 'img' ? (
+        React.cloneElement(thumbnail as JSX.Element, {
+          className: 'iui-tile-thumbnail-picture',
+        })
+      ) : React.isValidElement(thumbnail) ? (
+        React.cloneElement(thumbnail, {
+          className: cx('iui-thumbnail-icon', thumbnail.props.className),
+        })
+      ) : (
+        thumbnail
+      )}
+      {leftIcon &&
+        React.cloneElement(leftIcon as React.ReactElement, {
```

Comment:
most of these dom elements should be subcomponents too. I expect the final markup to look something like:
```
<Tile>
  <Tile.ThumbnailArea>
    <Tile.Thumbnail />
    <Tile.QuickAction>⭐️</Tile.QuickAction>
    <Tile.QuickAction>💬</Tile.QuickAction>

    <Tile.Badges>
     <Badge>badge 1</Badge>
     <Badge>badge 2</Badge>
    </Tile.Badges>
  </Tile.ThumbnailArea>

  <Tile.Name>Hello</Tile.Name>

   ...
</Tile>
```

in general, we want to avoid `cloneElement` and prefer making new subcomponents

Reply:
I'm unsure how we can avoid `cloneElement` in cases where the Element supplied by user have many props or components within it. For example a thumbnail (this is made up, not real use of these components):
```
    <Avatar
      size='x-large'
      status='online'
      abbreviation='TR'
      backgroundColor={getUserColor('Terry Rivers')}
      image={
        <img src='https://itwinplatformcdn.azureedge.net/iTwinUI/user-placeholder.png' />
      }
      title='Terry Rivers'
        <IconButton styleType='borderless'>
            <SvgInfo />
        </IconButton>
    />
```
And this could be something the user made just for the thumbnail picture, which I think `cloneElement` is the best use here anyway?

Reply:
So based on the above, my thumbnail subcomponent now looks like this:

```
const TileThumbnail = React.forwardRef((props, ref) => {
  const { as: Element = 'div', className, thumbnail, ...rest } = props;
  return (
    <Element
      className={cx('iui-tile-thumbnail', className)}
      ref={ref}
      {...rest}
    >
      {thumbnail && typeof thumbnail === 'string' ? (
        <div
          className='iui-tile-thumbnail-picture'
          style={{ backgroundImage: `url(${thumbnail})` }}
        />
      ) : thumbnail && (thumbnail as JSX.Element).type === 'img' ? (
        React.cloneElement(thumbnail as JSX.Element, {
          className: 'iui-tile-thumbnail-picture',
        })
      ) : React.isValidElement(thumbnail) ? (
        React.cloneElement(thumbnail, {
          className: cx('iui-thumbnail-icon', thumbnail.props.className),
        })
      ) : (
        thumbnail
      )}
    </Element>
  );
}) as PolymorphicForwardRefComponent<'div', TileThumbnailOwnProps>;
```

Do you think it should be broken down even more? But either ways where it is a custom element, there would be a `cloneElement` regardless, no?

Reply:
So even if we make a new subcomponent for QuickAction, it would also return a `cloneElement` of whatever is passed in, like how we have it originally. Or if you can show me an example of how to break that down without cloning? a use case: 
```
      <Tile.QuickAction
          leftIcon={
            <IconButton styleType='borderless'>
              <SvgInfo />
            </IconButton>
          }
       />
```

Reply:
Turned everything into subcomponents